src/app/App.js

```diff
@@ -4,11 +4,14 @@ import Logger from '../utils/Logger'
 import decodeLogs from '../helpers/decodeLogs'
 import encodeCall from '../helpers/encodeCall'
 
+import AppProvider from './AppProvider'
+import AppDeployer from './AppDeployer'
+
 const log = new Logger('App')
 
-export default class AppWrapper {
-  constructor(app, factory, appDirectory, _package, version, txParams = {}) {
-    this.app = app
+export default class App {
+  constructor(_app, factory, appDirectory, _package, version, txParams = {}) {
+    this._app = _app
```

Comment:
Something I kept thinking after the original review: why hiding the wrapped App contract now, when we are still exposing all dependent contracts (package, factory, and directories)? On the other hand, the inner contract does not seem to be used anywhere (just checked the CLI), so it'd be ok to remove.